RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -3391,6 +3405,19 @@ float [stat] provideStats(int [stat] amt, boolean doEquips, boolean speculative)
 	{
 		foreach eff in effects
 		{
+			boolean effectMatters = false;
+			foreach st in amt
+			{
+				if(!pass(st) && (numeric_modifier(eff, st) > 0 || numeric_modifier(eff, st + " Percent") > 0))
+				{
+					effectMatters = true;
+				}
+			}
+			if(!effectMatters)
+			{
+				auto_log_debug("Skipping effect " + eff + " because it has no relevant stats");
```

Comment:
Does this code run frequently? I'm a bit worried about logspam, since "I skipped the +muscle buff since I'm looking for +moxie" isn't a super useful debug message. Looks great other than that.

Reply:
Hmm yeah that's a good point, I'm going to cull the skipping effect print for stats and resists.